tools/release_do_nothing.py

```diff
@@ -0,0 +1,758 @@
+        _wait_for_done(message)
+
+    message = (
+        "Commit and push all the What's New changes.\n"
+        f'git commit -am "Whats new updates for {release_strings.tag} .";\n'
+        f"git push -u origin {working_branch};"
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Follow the Pull Request process to get {working_branch} "
+        f"merged into upstream/{release_strings.branch} .\n"
+        "Make sure the documentation is previewed during this process."
+    )
+    _wait_for_done(message)
+
+    return rsts
+
+
+def cut_release(
+    release_strings: ReleaseStrings, is_release_candidate: bool
+) -> None:
+    _break_print("The release ...")
+
+    message = (
+        "Visit https://github.com/SciTools/iris/releases/new to start "
+        "drafting a new release."
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Select {release_strings.branch} as the Target.\n"
+        f"Input {release_strings.tag} as the new tag to create, and also as "
+        "the Release title."
+    )
+    _wait_for_done(message)
+
+    message = (
+        "Craft an appropriate release description in the main text box.\n"
+        "Be sure to mention the What's New entry, conda-forge and PyPI - you "
+        "will need to return later to make these into links.\n"
+        "Be careful to change the appropriate words if copying from a "
+        "previous release description."
+    )
+    _wait_for_done(message)
+
+    if is_release_candidate:
+        message = (
+            "This is a release candidate - include the following instructions "
+            "for Conda-installing:\n"
+            "conda install -c conda-forge/label/rc_iris iris;"
+        )
+        _wait_for_done(message)
+
+        message = (
+            "This is a release candidate - tick the box to set this as a "
+            "pre-release."
+        )
+        _wait_for_done(message)
+
+    message = "Tick the box to set this as the latest release."
+    _wait_for_done(message)
+
+    message = "Click: Publish release !"
+    _wait_for_done(message)
+
+    message = (
+        "The CI will now run against this new tag, including automatically "
+        "publishing to PyPI."
+    )
+    _break_print(message)
+
+
+def check_rtd(
+    release_strings: ReleaseStrings, is_release_candidate: bool
+) -> None:
+    _break_print("Read the Docs checks ...")
+
+    message = (
+        "Visit https://readthedocs.org/projects/scitools-iris/versions/ and "
+        "make sure you are logged in."
+    )
+    _wait_for_done(message)
+
+    message = f"Set {release_strings.tag} to Active, un-Hidden."
+    _wait_for_done(message)
+
+    message = f"Set {release_strings.branch} to Active, Hidden."
+    _wait_for_done(message)
+
+    message = (
+        "Consider deactivating older branch doc builds - those formatted 0.0.x"
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Visit https://scitools-iris.readthedocs.io/en/{release_strings.tag} "
```

Comment:
Perhaps add disclaimer; docs might take some time to build.

Reply:
0b9fd768f